src/plugins/controls/public/control_group/control_group_strings.ts

```diff
@@ -10,6 +10,21 @@ import { i18n } from '@kbn/i18n';
 import { RANGE_SLIDER_CONTROL } from '../range_slider';
 
 export const ControlGroupStrings = {
+  invalidControlWarning: {
+    title: i18n.translate('controls.controlGroup.invalidControlWarning.title', {
+      defaultMessage: 'Selections are returning no results',
+    }),
+    text: i18n.translate('controls.controlGroup.invalidControlWarning.text', {
+      defaultMessage:
+        'Some control selections are returning no results. Remove them for complete results.',
+    }),
+    dismissButtonLabel: i18n.translate(
+      'controls.controlGroup.invalidControlWarning.dismissButtonLabel',
+      {
+        defaultMessage: 'Do not show again',
+      }
+    ),
+  },
```

Comment:
Agreed. Additionally, I would add a color to it, either primary or warning. If warning, we can add the alert icon.

Reply:
Also, the button in black seems out of place? Not only style wise but it's just atypical in an **EuiToast**. Can't we show the toast once and get rid of the button?

Reply:
@andreadelrio Looks like the default design of the warning toasts uses the `help` icon rather than the `warning` - is that okay? This should be the same for all other warning toasts in Kibana that use the `notifications` service

![image](https://github.com/elastic/kibana/assets/8698078/d3f23273-96db-40d5-904a-b9df07360207)

Reply:
Resolving this in favour of https://github.com/elastic/kibana/pull/174201#issuecomment-1955274867